Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Interactable.cs

```diff
@@ -940,11 +939,11 @@ public void OnBeforeFocusChange(FocusEventData eventData)
 
         public void OnFocusEnter(FocusEventData eventData)
         {
-            Debug.Assert(focusingPointers.Count > 0,
```

Comment:
When I updated this assert before, it was specifically happening in cases where this script wasn't really being used "correctly" - i.e. this script assumes that it gets the full range of focus events (i.e. IMixedRealityFocusChangedHandler and IMixedRealityFocusHandler.

What this assert means is that, there's something else in the chain that is eating one of the interface events (or it just isn't getting the other set of events but is getting the others). It's possible though that I had misunderstood the calling conventions/requirements here, in which case it would probably be worth removing the assert if that's the case.